presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorDistributedQueries.java

```diff
@@ -145,4 +150,86 @@ public void testCreateBucketedTable()
 
         assertUpdate("DROP TABLE orders_bucketed");
     }
+
+    @Test
+    public void testTablesSystemTable()
+    {
+        assertUpdate("" +
+                "CREATE TABLE system_tables_none(t1 timestamp, o1 varchar, o2 double, b1 bigint, b2 bigint) " +
```

Comment:
The table name is "none" but it has a temporal column. The ones below say "temporal_bucketed", so this is a bit confusing / inconsistent.